DigitalLearningSolutions.Web.Tests/Controllers/LearningPortal/LearningPortalControllerTests.cs

```diff
@@ -0,0 +1,60 @@
+namespace DigitalLearningSolutions.Web.Tests.Controllers.LearningPortal

```

Comment:
This has just been moved and any tests relating to the error pages, terms or accessibility pages moved to the learning solutions controller tests 🙂